src/cogs/interactions/slash/mod.py

```diff
@@ -843,7 +842,7 @@ async def unlock_channel(
             )
             return
 
-        overwrites.send_messages = False
+        overwrites.send_messages = True

```

Comment:
it shouldn't be true either. you know that setting that exists in between the green tick and red cross? check image below. i want that
<img width="474" height="126" alt="image" src="https://github.com/user-attachments/assets/d874e948-60e5-4039-ad53-7505a8b4c79a" />

i think what you should be doing is removing the `send_messages` overwrite. i'm not entirely sure about it. let me know

Reply:
or maybe `None`. i'm not sure
check out how it happens in the JS bot here: https://github.com/sach-12/pesu-bot-js/blob/main/src/client/commands/mod.js#L480